config/prod/config.yaml

```diff
@@ -0,0 +1,5 @@
+ingestion_bucket_name: recover-ingestion
+input_bucket_name: recover-input-data
+cloudformation_artifact_bucket_name: recover-cloudformation
+intermediate_bucket_name: recover-intermediate-data
+processed_data_bucket_name: recover-processed-data
```

Comment:
Instead of including the bucket names in the stack group config we should list the bucket stacks as dependencies in the relevant config files and reference the bucket name using 
```
!stack_output_external bucket-stack::BucketName
```

Reply:
Based on our recent slack discussion, and to avoid redundancy in our stack dependencies with the buckets, we will use the `stack_group_config` way

Reply:
You'll also want to use a `stack_group_config` reference for the BucketName parameter in the bucket stacks themselves: `config/prod/s3-cloudformation-bucket.yaml`, `config/prod/s3-ingestion.yaml`, etc.

Reply:
Feel free to do the same in dev.

Reply:
Yup, I have a ticket that encompasses all of that here: https://sagebionetworks.jira.com/browse/ETL-401